src/wallet/api/wallet.cpp

```diff
@@ -1099,6 +1134,59 @@ bool WalletImpl::finalizeMultisig(const vector<string>& extraMultisigInfo) {
     return false;
 }
 
+bool WalletImpl::exportMultisigImages(string& images) {
+    try {
+        clearStatus();
+        checkMultisigWalletReady(m_wallet);
+
+        auto blob = m_wallet->export_multisig();
+        images = epee::string_tools::buff_to_hex_nodelimer(blob);
+        return true;
+    } catch (const exception& e) {
+        LOG_ERROR("Error on exporting multisig images: ") << e.what();
+        setStatusError(string(tr("Failed to export multisig images: ")) + e.what());
+    }
+
+    return false;
+}
+
+size_t WalletImpl::importMultisigImages(const vector<string>& images) {
+    try {
+        clearStatus();
+        checkMultisigWalletReady(m_wallet);
+
+        bool ready;
+        if (!m_wallet->multisig(&ready)) {
+            throw runtime_error("Wallet is not multisig");
+        }
+
+        if (!ready) {
+            throw runtime_error("Multisig wallet is not finalized yet");
```

Comment:
Seems spurious, since `checkMultisigWalletReady()` is called just above

Reply:
you absolutely right, thank you! i wrote this code first then refactored a bit and forgot about this lines. will fix it

Reply:
i deleted redundant checks but github shows it still for some reason